acceptance-tests/src/test/java/uk/gov/di/test/step_definitions/Login.java

```diff
@@ -304,4 +304,38 @@ public void theUserEntersTheSixDigitSecurityCodeFromTheirEmail() {
         }
         findAndClickContinue();
     }
+
+    @When("the user enters their {string} code")
+    @When("the new user enters their {string} code")
+    public void theUserEntersTheirAuthCode(String codeType){
+
+        if (codeType == "email") {
+            loginPage.enterSixDigitSecurityCode(System.getenv().get("TEST_USER_EMAIL_CODE"));
+        }
+        
+        if (codeType == "auth app") {
+            loginPage.enterSixDigitSecurityCode(System.getenv().get("TEST_USER_PHONE_CODE"));
+        }
+
+        findAndClickContinue();
+    }
+
+    @And("confirmation that the user will get security codes via {string} is displayed")
+    public void confirmationThatTheUserWillGetSecurityCodesViaIsDisplayed(String authenticationType){
+        assertEquals("You've changed how you get security codes", driver.getTitle());
```

Comment:
Are both these assertions necessary? Looks like they do the same thing?

Reply:
One checks the page title and the other checks the text on the page.